docs/Built-ins.md

```diff
@@ -13,6 +13,181 @@ These rules are available without any npm installation, via the `WORKSPACE` inst
 [labels]: https://bazel.build/docs/build-ref.html#labels
 
 
+## node_repositories
+
+To be run in user's WORKSPACE to install rules_nodejs dependencies.
+
+This rule sets up node, npm, and yarn.
+
+The versions of these tools can be specified in one of three ways:
+- Normal Usage:
+Specify no explicit versions. This will download and use the latest NodeJS & Yarn that were available when the
+version of rules_nodejs you're using was released.
+- Forced version(s):
+You can select the version of NodeJS and/or Yarn to download & use by specifying it when you call node_repositories,
+but you must use a value that matches a known version.
+- Using a custom version:
+You can pass in a custom list of NodeJS and/or Yarn repositories and URLs for node_resositories to use.
+- Using a local version:
+To avoid downloads, you can check in vendored copies of NodeJS and/or Yarn and set vendored_node and or vendored_yarn
+to point to those before calling node_repositories.
+
+This rule exposes the `@nodejs` workspace containing some rules the user can call later:
+
+- Run node: `bazel run @nodejs//:node path/to/program.js`
+- Install dependencies using npm: `bazel run @nodejs//:npm install`
```

Comment:
does this need to be `bazel run @nodejs//:npm -- install`?

Reply:
No, this is okay since there is no hyphen on the arg.
note this was just moved from lower in the file